includes/packages/Manuel-Shipment/manual-shipment.php

```diff
@@ -0,0 +1,297 @@
+
+class Hez_Shipping_Tracking
+{
+	public $shipping_carriers;
+	public function __construct()
+	{
+		// Shipping Carriers Initialization
+
+		$this->shipping_carriers=[
+							'TR_YURTICI' => 
+								[
+								'name'=>'Yurtiçi Kargo',
+								'logo'=>'http://placehold.it/120x120&text=image2',
+								'tracking_link'=>'http://www.fedex.com/Tracking?tracknumbers=',
+								'is_available'=>true
+								],
+							'TR_MNG' => [
+								'name'=>'MNG Kargo',
+								'logo'=>'http://placehold.it/120x120&text=image3',
+								'tracking_link'=>'http://www.ups.com/WebTracking/track?loc=en_US/',
+								'is_available'=>true
+								],
+							'TR_ARAS' => [
+								'name'=>'Aras Kargo',
+								'logo'=>'http://placehold.it/120x120&text=image4',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>true
+								],
+							'TR_UPS' => [
+								'name'=>'UPS Kargo',
+								'logo'=>'http://placehold.it/120x120&text=image1',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>true
+								],
+							'TR_SURAT' => [
+								'name'=>'Sürat Kargo',
+								'logo'=>'http://placehold.it/120x120&text=image2',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>true
+								],
+							'TR_HOROZ' => [
+								'name'=>'Horoz Loijistik',
+								'logo'=>'http://placehold.it/120x120&text=image3',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								],
+							'TR_KARGO_TURK' => [
+								'name'=>'Kargo Türk',
+								'logo'=>'http://placehold.it/120x120&text=image4',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								],
+							'DHL' => [
+								'name'=>'DHL',
+								'logo'=>'http://placehold.it/120x120&text=image1',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								],
+							'TNT' => [
+								'name'=>'TNT',
+								'logo'=>'http://placehold.it/120x120&text=image2',
+								'tracking_link'=>'http://www.usps.com/shipping/trackandconfirm/',
+								'is_available'=>false
+								]
+							];
+
+		$this->shipping_carriers = $this->get_shipping_providers();
+
+		// Action to register custom order status
+		add_action( 'init', [$this,'hez_register_shipped_order_status'] );
+
+		// Add custom status to order status list
+		add_filter( 'wc_order_statuses', [$this,'hez_add_shipped_to_order_statuses'] );
+
+		// Add Custom fields for woocommerce order
+		add_action( "add_meta_boxes", [$this,"hez_add_meta_boxes"] );
+		
+		// Ajax handler function
+		add_action( 'wp_ajax_hezarfen_shipping_data_action', [$this,'hezarfen_shipping_data_action'] );
+
+		// Add Custom fields for woocommerce order
+		add_action( 'wp_ajax_nopriv_hezarfen_shipping_data_action', [$this,'hezarfen_shipping_data_action'] );
```

Comment:
we don't need it since we don't have support for not logged-in users, please remove this line.

https://developer.wordpress.org/reference/hooks/wp_ajax_action/